### PR TITLE
Add --content switch to just show the term content text.

### PR DESCRIPTION
**Pull request by cmars**
I don't know if it's pretty, but it's pretty useful.

### CODE REVIEW

File: cmd/show_term.go [hunk 1]
```diff
@@ -105,7 +107,11 @@ func (c *showTermCommand) Run(ctx *cmd.Context) error {
 		return errors.Trace(err)
 	}
 
-	err = c.out.Write(ctx, response)
+	if c.ShowContent {
+		_, err = ctx.Stdout.Write([]byte(response.Content))
```
Comment (alesstimec) [inline cmd/show_term.go]: can you please add a test for this case?

**Comment by alesstimec** [comment]
LGTM once the test is added

File: cmd/show_term.go (hunk 1 above)
Comment (cmars) [inline cmd/show_term.go]: Will do!

**Comment by cmars** [comment]
$$merge$$

**Pull request merged**